protos/log_files/log_files.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package mavsdk.rpc.log_files;
+
+import "mavsdk_options.proto";
+
+option java_package = "io.mavsdk.log_files";
+option java_outer_classname = "LogFilesProto";
+
+// Allow to download log files from the vehicle after a flight is complete.
+// For log streaming during flight check the logging plugin.
+service LogFilesService {
+    // Get List of log files.
+    rpc GetEntries(GetEntriesRequest) returns(GetEntriesResponse) {}
+    // Download log file.
+    rpc SubscribeDownloadLogFile(SubscribeDownloadLogFileRequest) returns(stream DownloadLogFileResponse) {
+        option (mavsdk.options.async_type) = ASYNC;
+        option (mavsdk.options.is_finite) = true;
+    }
+}
+
+message GetEntriesRequest {}
+message GetEntriesResponse {
+    LogFileResult log_file_result = 1;
+    repeated Entry entries = 2; // List of entries
+}
+
+message SubscribeDownloadLogFileRequest {
+    uint32 id = 1; // ID of the log file to download
+    string path = 2; // Path of where to download log file to.
+}
+message DownloadLogFileResponse {
+    LogFileResult log_file_result = 1;
+    float progress = 2; // Progress if result is progress
```

Comment:
This assumes that we will never return anything other than this `float`. Fine for me, but just to make sure :-)

Reply:
I would think so, unless download speed :thinking: ...

Reply:
I'm changing it to a `ProgressData`.